src/java.base/share/classes/jdk/internal/classfile/impl/ClassPrinterImpl.java

```diff
@@ -497,7 +497,7 @@ private static Node[] elementValueToTree(AnnotationValue v) {
             case OfBoolean cv -> leafs("boolean", String.valueOf((int)cv.constantValue() != 0));
             case OfClass clv -> leafs("class", clv.className().stringValue());
             case OfEnum ev -> leafs("enum class", ev.className().stringValue(),
-                                    "contant name", ev.constantName().stringValue());
+                                    "constant name", ev.constantName().stringValue());
```

Comment:
Here the same typo is repeated in ClassPrinterTest, which actually fails after the fix.

Reply:
Thanks for the reminder. I've checked IDE-reported typos in the tests and fixed 2; jdk.classfile tests pass on my machine now.